web/impact/requirements/base.txt

```diff
@@ -34,7 +34,7 @@ Django==1.10.3
 djangorestframework
 drf-schema-adapter
 drf-tracking
-git+https://github.com/masschallenge/django-accelerator.git@2017.09.06a#egg=django-accelerator
+git+https://github.com/masschallenge/django-accelerator.git@AC-4812#egg=django-accelerator
 
 # django-configurations from Git master (needed for Django 1.8 support)
```

Comment:
it looks like 'simpleuser' has been removed from this app. is it now packaged in with django-accelerator or is it its own standalone app (in which case we would need to add it to requirements.txt)

Reply:
The code comes from django-accelerator.  If we need to add 'simpleuser' then we would also need to add 'accelerator'.  It's not clear to me if there is a real issue here or not.  It certainly all seems to be working as it is.